qcodes/utils/val_mapping.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict
+from typing import Any, Dict, Tuple, Union
+
+
+def create_on_off_val_mapping(
```

Comment:
we could move this util into parameters module because it's used only there and is relevant for paramters only really. that would reduce the size of utils a bit :)